packages/kit/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@sveltejs/kit",
 	"version": "2.5.0",
-	"description": "The fastest way to build Svelte apps",
+	"description": "SvelteKit, the fastest way to build Svelte apps",
```

Comment:
I've also considered using `-`. Would love more opinions on this.

I've got the comma expression from the SK2 announcement.

> Version 2.0 of [SvelteKit](https://kit.svelte.dev/), the official framework for building apps with Svelte, is now available.

Also, it would be nice to update the description itself - maybe include that it is the official framework? This would require updating the README as well.